terragrunt/aws/cloudfront/cloudfront.tf

```diff
@@ -0,0 +1,55 @@
+resource "aws_cloudfront_origin_access_identity" "origin_access_identity" {
+  comment = "cloudfront origin access identity"
+}
+
+resource "aws_cloudfront_distribution" "distribution" {
+  for_each            = var.s3_bucket_regional_domain_name
+  default_root_object = "index.html"
```

Comment:
Could you add a price class and alias section here.  The price class lets you determine how many CloudFront edge locations are used, so I'd go for `All` since we want this to be snappy all over the world.

For the alias, it's what lets people access the CloudFront distribution using one of the domains like `digital.canada.ca` or `numerique.canada.ca`.  

Here's an example of what that looks like:
https://github.com/cds-snc/gcds-terraform/blob/1ffb8d00f6b34da5d9d8eec9f262d1abbe96f56b/terragrunt/aws/cdn/cloudfront.tf#L7-L8